torch/_subclasses/functional_tensor.py

```diff
@@ -17,6 +17,13 @@
 not_implemented_log = torch._logging.getArtifactLogger(__name__, "not_implemented")
 
 
+def _conversion_method_template(**extra_kwargs):
```

Comment:
@zhxchen17 a big comment describing why this code lives in `functional_tensor.py` would be nice. Something along the lines of:

- export doesn't like ops that are maybe-aliasing (like `aten.to()`), so it hooks off of functionalization to change `.to()` under export to be always-copy, and error on programs for this this transformation will change semantics
- there are a bunch of tensor methods that have similar behavior, like .float() or .cpu(). In theory, we should get the same handling "for free" if they all were implemented by calling into `at::to()`, but they do not call into `at::to()` in eager mode for to minimize framework overhead. So instead, we are forcing them to go through `at::to()` directly in `functional_tensor.py` so they get the same treatment